### PR TITLE
implemented close recipe button

### PR DESCRIPTION
**Pull request by lyudmyla-horbyk**
#25

### CODE REVIEW

File: src/views/RecipeItem.vue [hunk 1]
```diff
@@ -67,6 +69,9 @@ export default {
     editRecipe(editedRecipe) {
       recipes.splice(this.$route.params.index, 1, editedRecipe);
       this.recipe = recipes[this.$route.params.index];
+    },
+    closeRecipe() {
```
Comment (probil) [inline src/views/RecipeItem.vue]: Remove this function. It's seem to be redundant

File: src/views/RecipeItem.vue [hunk 2]
```diff
@@ -5,7 +5,9 @@
     </div>
     <div class="fullsize-recipe-container-details">
       <a href="#">
-        <i class="fas fa-times"></i>
+        <router-link :to="{ name: 'Home' }"
```
Comment (probil) [inline src/views/RecipeItem.vue]: `<router-link>` is converted to `<a>` by vue.js under the hood.
There is not need for `<a/>` in `<a/>`. Remove it please

File: src/views/RecipeItem.vue [hunk 3]
```diff
@@ -4,11 +4,10 @@
       <img :src="recipe.url" :alt="recipe.title" />
     </div>
     <div class="fullsize-recipe-container-details">
-      <a href="#">
-        <router-link :to="{ name: 'Home' }"
-          ><i v-on:click="closeRecipe" class="fas fa-times"></i
-        ></router-link>
-      </a>
+      <router-link :to="{ name: 'Home' }"
+        ><i v-on:click="closeRecipe" class="fas fa-times"></i
```
Comment (probil) [inline src/views/RecipeItem.vue]: You need to remove `v-on:click="closeRecipe"` since it's not more needed

Comment (lyudmyla-horbyk) [reply src/views/RecipeItem.vue]: @probil Done!

**Pull request merged**